src/Appwrite/Platform/Modules/Proxy/Http/Rules/Function/Create.php

```diff
@@ -134,49 +90,26 @@ public function action(string $domain, string $functionId, string $branch, Respo
         $deployment = $dbForProject->getDocument('deployments', $function->getAttribute('deploymentId', ''));
 
         // TODO: (@Meldiron) Remove after 1.7.x migration
-        $isMd5 = System::getEnv('_APP_RULES_FORMAT') === 'md5';
-        $ruleId = $isMd5 ? md5($domain->get()) : ID::unique();
-
-        $status = 'created';
-        if (\str_ends_with($domain->get(), $functionsDomain) || \str_ends_with($domain->get(), $sitesDomain)) {
-            $status = 'verified';
-        }
-        if ($status === 'created') {
-            $validators = [];
-            $targetCNAME = new Domain(System::getEnv('_APP_DOMAIN_TARGET_CNAME', ''));
-            if ($targetCNAME->isKnown() && !$targetCNAME->isTest()) {
-                $validators[] = new DNS($targetCNAME->get(), Record::TYPE_CNAME);
-            }
-            if ((new IP(IP::V4))->isValid(System::getEnv('_APP_DOMAIN_TARGET_A', ''))) {
-                $validators[] = new DNS(System::getEnv('_APP_DOMAIN_TARGET_A', ''), Record::TYPE_A);
-            }
-            if ((new IP(IP::V6))->isValid(System::getEnv('_APP_DOMAIN_TARGET_AAAA', ''))) {
-                $validators[] = new DNS(System::getEnv('_APP_DOMAIN_TARGET_AAAA', ''), Record::TYPE_AAAA);
-            }
-
-            if (empty($validators)) {
-                throw new Exception(Exception::GENERAL_SERVER_ERROR, 'At least one of domain targets environment variable must be configured.');
-            }
-
-            $validator = new AnyOf($validators, AnyOf::TYPE_STRING);
-            if ($validator->isValid($domain->get())) {
-                $status = 'verifying';
-            }
-        }
+        $ruleId = System::getEnv('_APP_RULES_FORMAT') === 'md5' ? md5($domain) : ID::unique();
 
         $owner = '';
         if (
-            ($functionsDomain != '' && \str_ends_with($domain->get(), $functionsDomain)) ||
-            ($sitesDomain != '' && \str_ends_with($domain->get(), $sitesDomain))
+            ($functionsDomain != '' && \str_ends_with($domain, $functionsDomain)) ||
+            ($sitesDomain != '' && \str_ends_with($domain, $sitesDomain))
```

Comment:
Let's open all create rules files side by site (api, function, redirect, site), and compare them. Ideally their logic is identical; I think this is not same as we havein API createRule. Let's compare entire file, only keeping differences that are on purpose (having comment for those differences would be great too)